TrafficCapture/captureOffloader/src/test/java/org/opensearch/migrations/trafficcapture/StreamChannelConnectionCaptureSerializerTest.java

```diff
@@ -155,14 +156,41 @@ public void testBasicDataConsistencyWhenChunking() throws IOException, Execution
         Assertions.assertEquals(packetData, reconstructedData);
     }
 
+    @Test
+    public void testCloseAfterWriteWillFlushWhenSpaceNeeded() throws IOException, ExecutionException, InterruptedException {
```

Comment:
nit: "testCloseObservation..." would be clearer.  Sounds like you might have meant to close a stream